shell.nix

```diff
@@ -1,67 +1,22 @@
+# This derivation assumes a toplogy file where the following attributes are
+# defined:
+#
+# - bftCoreNodes
```

Comment:
bftCoreNodes / stakePoolNodes not needed anymore (using `stakePool` discriminant instead)